### PR TITLE
Master - upgrade EF Core to 3.1.3 and CsvHelper to 15.0.4

### PR DESCRIPTION
**Pull request by npagare**
A PR to upgrade EF Core to 3.1.3 and CsvHelper to 15.0.4
All test cases were run successfully.

I would appreciate if you accept this PR.
Thank you.

### CODE REVIEW

File: EFCore.Seeder.Console/Program.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            var csvConfiguration = new CsvHelper.Configuration.Configuration
+            var csvConfiguration = new CsvHelper.Configuration.CsvConfiguration(new System.Globalization.CultureInfo("en-US")) // .Configuration
```
Comment (DanielAltamirano) [inline EFCore.Seeder.Console/Program.cs]: Can you replace this for the following lines? It's a better practice to define the culture in a single place that's used throughout the code (even when it's not used anywhere else at the moment).

`CultureInfo.CurrentCulture = new CultureInfo("en-US");`

`var csvConfiguration = new CsvHelper.Configuration.CsvConfiguration(CultureInfo.CurrentCulture)`

**Comment by DanielAltamirano** [comment]
@IoTFier Looks good! Only one minor comment, so if you can fix that, I'm happy to merge your PR.
Thanks a lot for this!

**Review by DanielAltamirano — CHANGES_REQUESTED** [review]
Looks good! Only a minor comment, so if you can fix that, I'm happy to merge your PR.

Thanks a lot for this!

File: EFCore.Seeder.Console/Program.cs (hunk 1 above)
Comment (npagare) [reply EFCore.Seeder.Console/Program.cs]: I agree with you. Will do that and respond after changes are made and tested.

Comment (npagare) [reply EFCore.Seeder.Console/Program.cs]: Hi @DanielAltamirano, I have made recommended changes. All Test ran successfully.
Please take a look. 
Thank you.

**Review by DanielAltamirano — APPROVED** [review]

**Comment by DanielAltamirano** [comment]
@IoTFier Merged. Thanks, great work!

I'll publish a new nuget package.

**Comment by DanielAltamirano** [comment]
@IoTFier Package published. Cheers!

**Pull request merged**